source/change-streams/tests/change-streams.yml

```diff
@@ -297,3 +298,117 @@ tests:
           fullDocument:
             z:
               $numberInt: "3"
+  -
+    description: Test all change event types
+    minServerVersion: "3.6.0"
```

Comment:
I think you'll need to bump the minimum server version for some of these, which means you might need to break this test apart into a few individual tests.